src/styles/header.scss

```diff
@@ -0,0 +1,81 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+@import './stars';
+@import './colors';
+@import './card';
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+
+  --header-height: 60px;
```

Comment:
Custom properties (CSS variables) should be defined in a :root selector for global scope, or within an appropriate selector if they are meant to be scoped to specific parts of the page.